generators/app/templates/src/app/i18n/__raw.i18n-lang-selector.component.html

```diff
@@ -0,0 +1,5 @@
+<select (change)="setLanguage($event.target.value)">
+  <option *ngFor="let language of languages" [value]="language" [selected]="language === currentLanguage">
+    {{language | translate}}
```

Comment:
From a readability perspective, I believe that entries in a language selection dropdown should not be translated. For instance, if I can not read Chinese, and every entry in the list is written in Chinese, then I will never manage to find my own language in the list.

(this comment applies to all four components)

Reply:
No, the list should display the language name with the localize name.

```html
<div aria-labelledby="language-dropdown" ngbdropdownmenu="" class="dropdown-menu show" x-placement="bottom-right" style="top: 40px; left: -50px;">
    <!--bindings={
  "ng-reflect-ng-for-of": "en-US,fr-FR,zh-CN"
}--><button class="dropdown-item">
      English (US)
    </button><button class="dropdown-item">
      Français
    </button><button class="dropdown-item">
      简体中文
    </button>
  </div>
```

Reply:
The result is good. I just realized it comes from i18n-locales.ts, so it's all defined in one place with different languages in the same file, which is a godo solution for this use case.